src/components/responsive-flyout/ResponsiveFlyout.scss

```diff
@@ -0,0 +1,32 @@
+@import '../../styles/variables';
+
+.bdl-OverlayHeader {
```

Comment:
Should these styles should live in `OverlayHeader.scss`?

Reply:
Or should it be a hybrid of this and what we had before, just with slightly less specificity?

```
.bdl-ResponsiveFlyout {
    .bdl-OverlayHeader {
        display: none;
    }
}